Evaluate-ModelarDB-Compression/main.py

```diff
@@ -0,0 +1,281 @@
+""" Script for evaluating ModelarDB with different data sets. """
+
+
```

Comment:
Maybe run a formatting tool on the file. I think there should only be a single newline between the module doc comment and imports.

Reply:
The file was already formatted using [black](https://github.com/psf/black), so I manually removed one of the empty lines from `Evaluate-ModelarDB-Compression/main.py` and `Evaluate-ModelarDB-Changes/main.py`.